src/racing/render_courses.c

```diff
@@ -1259,6 +1259,10 @@ void render_big_donut(struct UnkStruct_800DC5EC* arg0) {
     gSPDisplayList(gDisplayListHead++, ((uintptr_t) 0x07000230));
 }
 
+/**
+ * @brief dev made two version of the Course one for 1-4p and another for credit.
```

Comment:
How about something like this:
> The game has two versions of each course. One for normal gameplay and another for credits.
> The credits version is not split into segments so that the camera can view the whole course.

Reply:
Will do it later today